### DISCUSSION TITLE
How to deploy markdown files in quick start in the document in vercel？

### DISCUSSION THREAD

**Discussion by FeClickEngineer** [remix-run/remix#2000] [Q&A] [answered]
https://remix.run/docs/en/v1/tutorials/blog#pulling-from-a-data-source
There is such a piece of code in the document
```javascript
// relative to the server output not the source!
const postsPath = path.join(__dirname, "..", "posts");
```
However, the code cannot query the folder in the local environment
I changed it so that I can access it in the local environment

```javascript
const postsPath = path.resolve("posts")
```
However, neither of these two methods can query the folder of markdown files on vercel

After I built it locally, I found that the markdown file was not packaged. How can I solve this

**Comment by ccssmnn** [comment, marked as answer]
Using the file system is not supported in Vercel serverless functions. In the [Remix Discord Vercel Channel](https://discord.com/channels/770287896669978684/778004202282287104) (good resource) some people report that even using this [Vercel Guide](https://vercel.com/support/articles/how-can-i-use-files-in-serverless-functions) they cannot make it work.

You can store your files in an external database (e.g. S3 oder Cloud Storage) or use the [GitHub SDK](https://github.com/octokit/octokit.js) to fetch the files from GitHub.

**Reply by FeClickEngineer** [reply]
Thank you for your answer
